pkgs/top-level/python-packages.nix

```diff
@@ -4611,7 +4611,7 @@ in {
 
   offtrac = callPackage ../development/python-modules/offtrac { };
 
-  openpyxl = if isPy3k then
+  openpyxl = if (pythonAtLeast "3.6") then
```

Comment:
```suggestion
  openpyxl = if pythonAtLeast "3.6" then
```

Reply:
used to doing this with `lib.optionals`, which needs the parenthesis.

thanks